src/main/java/org/randywebb/wlts/beans/Assignment.java

```diff
@@ -4,20 +4,17 @@
 import java.util.Collections;
 import java.util.List;
 
-import org.randywebb.wlts.beans.AbstractBean;
-import org.randywebb.wlts.beans.Visit;
-
 import org.json.simple.JSONArray;
 import org.json.simple.JSONObject;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+//import org.slf4j.Logger;
+//import org.slf4j.LoggerFactory;
 
 /** Represents a ministering assignment. */
 public class Assignment extends AbstractBean {
 
 	/** In case we want to log something */
-	private static Logger log = LoggerFactory.getLogger(Assignment.class);
+	//private static Logger log = LoggerFactory.getLogger(Assignment.class);
```

Comment:
I commented out the loggers in any class that did not use them. I left them in there so they would be easy to add in once some logging is needed. I could add a trace in the constructor or something minimal in each class to make sure it gets used. Or I could investigate if I could disable the error for unused and see if I can have it ignore patterns.

Reply:
It is OK to comment them out. I think you have the right idea.
An unused private entity will trigger a compile-time warning, so I see the justification in commenting them out.
I think it is better to leave them commented out than to create a log statement just to remove the compile warning.
Best is to add logging where it can be helpful, but not to artificially add things just to justify having the Logger instance active.